test/test_utils/test_utils.dart

```diff
@@ -0,0 +1,80 @@
+import "package:kisgeri24/data/models/route.dart";
+import "package:kisgeri24/data/dto/route_dto.dart";
+import "package:kisgeri24/data/dto/route_equipment.dart";
+import 'package:kisgeri24/data/models/user.dart';
+import 'package:kisgeri24/data/models/wall.dart';
+import 'package:kisgeri24/data/dto/user_dto.dart';
+import "package:kisgeri24/data/models/year.dart";
+
+final Year testYear = Year.all(
+  "Some Year ID",
+  2024,
+  "Some Tenant ID",
+  1234567890,
+  1234567891,
+);
+
+final UserDto testUserAsDto = UserDto(
+  testUser.email,
+  testUser.firstClimberName,
+  testUser.secondClimberName,
+  testUser.userID,
+  testUser.teamName,
+  testUser.category,
+  testUser.appIdentifier,
+  testUser.startTime,
+  testUser.tenantId,
+  testUser.yearId,
+  testUser.enabled,
+);
+
+final User testUser = User(
+  email: "someUser@email.com",
+  category: "24H",
+  firstClimberName: "First Test Climber",
+  secondClimberName: "Second Test Climber",
+  teamName: "Awesome Test Team",
+  tenantId: "Some More Awesome Tenant ID",
+  userID: "Even More Awesome User ID",
+);
+
+class TestUtils {
+  static List<RouteDto> createRouteDto({int quantity = 1}) {
+    List<RouteDto> routes = [];
+    for (int i = 0; i < quantity; i++) {
+      routes.add(new RouteDto(
+        "route-$i",
+        "6b",
+        30,
+        RouteEquipment.bolted,
```

Comment:
```suggestion
        RouteEquipment.values[i%RouteEquipment.values.length],
```

Reply:
I wouldn't, because if the quantity would be large and would make the remainder more than the amount of enum entries in the RoutEquipment, then a single test run would cause an index out-of-bound error